crates/engraver/src/render/ir/element.rs

```diff
@@ -27,6 +35,11 @@ impl<T> Element<T> {
             Element::Polygon(polygon) => polygon.max_x(),
             Element::Symbol(symbol) => symbol.max_x(),
             Element::Text(text) => text.max_x(),
+            Element::Group { elements, .. } => elements
+                .iter()
+                .map(|e| e.max_x())
+                .reduce(|a, b| if a > b { a } else { b })
+                .unwrap(),
```

Comment:
And same here too

Reply:
Could this use `max_by` like `Polygon::max_x`?

Reply:
Yeah, max_by works great. Did not know about that.